### PR TITLE
Md4c switch

### PR DESCRIPTION
**Pull request by Waqar144**

### CODE REVIEW

File: src/entities/note.cpp [hunk 1]
```diff
@@ -2138,9 +2155,7 @@ QString Note::textToMarkdownHtml(QString str, const QString& notesPath,
     QString codeBackgroundColor = darkModeColors ? QStringLiteral("#444444") : QStringLiteral("#f1f1f1");
 
     // do some more code formatting
-    // the "pre" styles are for the full-width code block background color
     codeStyleSheet += QString(
-                    "pre { display: block; background-color: %1 } "
```
Comment (pbek) [inline src/entities/note.cpp]: I don't think you want to remove that, do you?

Comment (Waqar144) [reply src/entities/note.cpp]: Nah, leftover. I will fix that.

File: src/entities/note.cpp [hunk 2]
```diff
@@ -2138,7 +2135,6 @@ QString Note::textToMarkdownHtml(QString str, const QString& notesPath,
     QString codeBackgroundColor = darkModeColors ? QStringLiteral("#444444") : QStringLiteral("#f1f1f1");
 
     // do some more code formatting
-    // the "pre" styles are for the full-width code block background color
```
Comment (pbek) [inline src/entities/note.cpp]: can you please leave this in too

Comment (Waqar144) [reply src/entities/note.cpp]: ok :grin:

**Comment by pbek** [comment]
This is strange:

image

    
```
    .boton2 
    {
        background: #898989;
        border:0px;
        color:White;
        border-radius: 3px;
        padding: 6px 20px;
        font-size: 14px;
    }
    .boton2:focus 
    {
        outline:0px;
    }
    .boton2:hover 
    {
        background: #6E6E6E;   
    }
```

**Comment by pbek** [comment]
Damn, it's the same with hoedown. Another code block regression.

**Comment by Waqar144** [comment]
> Damn, it's the same with hoedown. Another code block regression, I updated the issue.

Yeah. Just fixed though.

It was an oversight on my part.

**Pull request merged**